configs/next/build.mk

```diff
@@ -0,0 +1 @@
+../11.0/build.mk
```

Comment:
Why not point to ../12.0/build.mk?

Reply:
It should.... It seems `make clone` has a bug.